CHANGELOG.md

```diff
@@ -5,10 +5,14 @@
 ### Changed
 
 - Always enable verbose (`-v`) flag when a log directory is configured (`coder.proxyLogDir`).
+- Replaced SSE paths with a one-way WebSocket and centralized creation on `OneWayWebSocket`, and unified socket handling.
```

Comment:
I think this is too implementation-specific to deserve a callout in a user-facing changelog.  Probably users do not particularly care if we use SSE or web sockets.  Since the functionality from the user perspective is identical in this case and there are no actions the user can take, we can just omit the change IMO.